addons/test_mail/tests/test_message_management.py

```diff
@@ -43,7 +44,10 @@ def test_mail_resend_workflow(self):
 
         # three more failure sent on bus, one for each mail in failure and one for resend
         self._reset_bus()
-        with self.mock_mail_gateway(), self.assertBus([(self.cr.dbname, 'res.partner', self.partner_admin.id)] * 3):
+        data = [(self.cr.dbname, 'res.partner', self.partner_admin.id)]
+        if self.user_partner != self.partner_admin:
```

Comment:
Is this condition necessary? Don't we know from the way the test is written if that's always true or false?